src/Content.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by TTND on 12/27/2016.
+ */
+import React from 'react'
+
+export default class Content extends React.Component {
+    constructor() {
```

Comment:
pass props in super as a best practice to avoid having this.props in constructor as undefined
```constructor(props){ super(props) ... } ```